Makefile

```diff
@@ -1,12 +1,13 @@
 NAME := selion
-VERSION := $(or $(VERSION),$(VERSION),'1.0.0')
+VERSION := $(or $(VERSION),$(VERSION),'develop')
 PLATFORM := $(shell uname -s)
-SELION_GRID_VERSION := $(or $(SELION_GRID_VERSION), $(SELION_GRID_VERSION), 'RELEASE')
-REPO := $(or $(REPO), $(REPO), 'releases')
-SELENIUM_VERSION := $(or $(SELENIUM_VERSION), $(SELENIUM_VERSION), '2.48')
-SELENIUM_FIX := $(or $(SELENIUM_FIX), $(SELENIUM_FIX), '2')
+SELION_GRID_VERSION := $(or $(SELION_GRID_VERSION), $(SELION_GRID_VERSION), '1.1.0-SNAPSHOT')
+REPO := $(or $(REPO), $(REPO), 'snapshots')
+SELENIUM_VERSION := $(or $(SELENIUM_VERSION), $(SELENIUM_VERSION), '2.53')
+SELENIUM_FIX := $(or $(SELENIUM_FIX), $(SELENIUM_FIX), '0')
```

Comment:
the defaults being the last release versions were intentional but if you're going to make this change, you can update the .travis.yaml too. I'm fine either way.